install/installer/trafodion_install

```diff
@@ -696,29 +625,30 @@ reservedTrafodionPorts="net.ipv4.ip_local_reserved_ports = ${trafodion_ports}"
 
 for node in $NODE_LIST
 do
-   redhatSystem=$(ssh $node -n '/sbin/sysctl net.ipv4.ip_local_reserved_ports | wc -l')
+   redhatSystem=$(ssh -q -n $node '/sbin/sysctl net.ipv4.ip_local_reserved_ports | wc -l')
    if [[ "$redhatSystem" -ge "1" ]]; then
-      reserved_ports=$(ssh $node -n '/sbin/sysctl net.ipv4.ip_local_reserved_ports' | awk -F'=' '{print $2;}' | sed 's/^ //')
-      ssh $node -n 'sudo chmod 777 /etc/sysctl.conf'
-      ssh $node -n 'sudo sed -i '/net.ipv4.ip_local_reserved_ports""\=*/d' /etc/sysctl.conf'
+      reserved_ports=$(ssh -q -n $node '/sbin/sysctl net.ipv4.ip_local_reserved_ports' | awk -F'=' '{print $2;}' | sed 's/^ //')
+      ssh -q -n $node 'sudo chmod 777 /etc/sysctl.conf'
+      ssh -q -n $node 'sudo sed -i '/net.ipv4.ip_local_reserved_ports""\=*/d' /etc/sysctl.conf'
 
    if [[ -z "${reserved_ports}" ]]; then
-      ssh $node -n 'sudo echo "$reservedTrafodionPorts" >> /etc/sysctl.conf'
+      ssh -q -n $node "sudo echo "$reservedTrafodionPorts" >> /etc/sysctl.conf"
    else
-      ssh $node -n "sudo echo "$reservedTrafodionPorts","$reserved_ports" >> /etc/sysctl.conf"
+      ssh -q -n $node "sudo echo "$reservedTrafodionPorts","$reserved_ports" >> /etc/sysctl.conf"
    fi
-   ssh $node -n 'sudo chmod 644 /etc/sysctl.conf'
-   ssh $node -n 'sudo /sbin/sysctl -p'
+   ssh -q -n $node 'sudo chmod 644 /etc/sysctl.conf'
+   ssh -q -n $node 'sudo /sbin/sysctl -p' 2>&1 > /dev/null
    else
-      echo "***ERROR: Unable to reserve DCS ports (${trafodion_ports})."
+      echo "***ERROR: Unable to reserve DCS ports (37800 and 40010)."
```

Comment:
Since the default ports changed (line 623) could you revert this change to utilize the trafodion_ports var ?